### ISSUE TITLE
http://jsonapi.org/schema returns the wrong content-type

### ISSUE THREAD

**Issue by MrTrick** [json-api/json-api#975]
`< Content-Type: application/octet-stream`

Surely it should be `Content-Type: application/json'?

Also, it might be beneficial to make "http://jsonapi.org/schema.json" return the schema as well.

**Comment by steveklabnik** [comment]
I don't think this is something that we can change, as it's GitHub Pages who serve these headers.

**Comment by bintoro** [comment]
Changing the filename as suggested will fix the media type.

**Comment by steveklabnik** [comment]
:+1: let's do it then

**Comment by vhenzl** [comment]
Please, do something with this. With wrong `Content-Type` header, other libs (e.g.   [justinrainbow/json-schema](https://github.com/justinrainbow/json-schema)) fail trying to load JSON:API schema.

**Comment by steveklabnik** [comment]
@vhenzl all it takes is writing a PR!

**Comment by vhenzl** [comment]
@steveklabnik, I'm happy to prepare a PR, but I think, there should be a discussion about appropriate solution first.

Possible ways I see:
1. Just rename `schema` to `schema.json`.  Quick but dirty. Will break all applications relying on the existence of http://jsonapi.org/schema URL. (Could affect e.g. #1079?)
2. Copy `schema` to `schema.json` and have both files. Backwards compatible, but will require maintaining two files.
3. Move the schema file from GitHub Pages and host it somewhere else as e.g. http://schema.jsonapi.org/v1. Possible to return correct `Content-Type` header without having `.json` extension, but will require to host somewhere and will probably need to move and maintain schema file out of this repository.

Also, this should consider how to work with versions and potentially more schemas (as mentioned in #867), if current URL is changed.

For 1) and 2) could be:

```
http://jsonapi.org/v1/schema.json
http://jsonapi.org/v1.1/request.json
http://jsonapi.org/v1.1/response.json
```

And for 3):

```
http://schema.jsonapi.org/v1
http://schema.jsonapi.org/v1.1/request
http://schema.jsonapi.org/v1.1/response
```

**Comment by ethanresnick** [comment]
@vhenzl Thanks for thinking through this! I think those are the right questions. My feeling is that we should do either option 2 or 3, as I don't want to break existing applications. In a perfect world, I'd say we should do 3, since I don't really want to have two files.

@tkellen I know you started working on a move to ec2 back in #671. Would you be up for finishing that and owning it? (Or maybe even just deploying to S3?)

**Comment by Relequestual** [comment]
We (JSON Schema) have the same problem.
We concluded that migrating to netlify and using their custom headers config would solve the problem, but we haven't actioned this.
